jest.conf.js

```diff
@@ -1,9 +1,11 @@
-module.exports = {
-  transformIgnorePatterns: ['/node_modules/(?!(keen-ui|epubjs)/).*/'],
+const baseConfig = require('kolibri-tools/jest.conf');
+
+module.exports = Object.assign(baseConfig, {
+  transformIgnorePatterns: ['/node_modules/(?!(keen-ui|epubjs|kolibri-tools|kolibri)/).*/'],
```

Comment:
maybe add a comment to this?

it's unclear to me why we'd be ignoring `kolibri-tools` and `kolibri`

Reply:
Can do - but this line is saying don't ignore these for code transpilation.

Reply:
ah, missed the `!`